### PR TITLE
dateutil: __version__ fix

### PR DESCRIPTION
**Pull request by iam-git**

### CODE REVIEW

**Comment by Begasus** [comment]
Checking the full recipe reveals a trailing whitespace at line 61? 
https://github.com/haikuports/haikuports/blob/6a0db88b92ffc75d69e05d1f323969e2aa12cbe5/dev-python/dateutil/dateutil-2.8.1.recipe#L61

**Comment by iam-git** [comment]
New releases of matplotlib check version of dateutil and raise ImportError if see something like "0.1.dev1+g6104a1a".

**Comment by korli** [comment]
Wouldn't it be better to use python-setuptools-scm?

**Comment by iam-git** [comment]
As I understand, python-setuptools-scm generates version number from git. In our case it is that empty git repo, made by haikuporter. 

So now I just set SETUPTOOLS_SCM_PRETEND_VERSION environment variable to get finally the right value.

File: dev-python/dateutil/dateutil-2.8.1.recipe [hunk 1]
```diff
@@ -45,6 +45,9 @@ REPLACES_python="
 	python_dateutil
 	"
 
+SETUPTOOLS_SCM_PRETEND_VERSION=$portVersion
+export SETUPTOOLS_SCM_PRETEND_VERSION
```
Comment (Begasus) [inline dev-python/dateutil/dateutil-2.8.1.recipe:49]: Stumbled upon a simular problem when checking up on zipp(_python), could you check if bumping setuptools to version 42.0.0 fixes it without this fix?

**Comment by Begasus** [comment]
@korli  @iam-git see here (zipp also had the version number wrong before I updated setuptools to 42.0.0)
https://github.com/jaraco/zipp/issues/53#issuecomment-627062475

**Comment by Begasus** [comment]
Ok, this indeed fixes the version format, checked with setuptools_scm vs setuptools, doesn't change anything, so setuptools should be ok to use here, while you are in the process of a revbump, maybe you could remove the python 2.7 dependency also ;)

**Comment by Begasus** [comment]
Looks like I need this fix also for iniconfig

**Comment by iam-git** [comment]
While I am in the process of a revbump, I would like to add python38 and python39 subpackages, but corresponding dependencies are not met.

**Comment by Begasus** [comment]
> While I am in the process of a revbump, I would like to add python38 and python39 subpackages, but corresponding dependencies are not met.

Most of the python recipes haven't been updated to use the newer python version, help welcome :)

**Comment by iam-git** [comment]
> Most of the python recipes haven't been updated to use the newer python version, help welcome :)

I'll consider revbumps for some packages I use, but at least in case of setuptools I'll rather rely to more skilled guys ;)

**Comment by Begasus** [comment]
Merged, seems like there are still some dependencies for python 2.7 that needs to be cleaned before we can remove it here

**Pull request merged**